src/Components/ContactEditForm/ContactEditForm.jsx

```diff
@@ -36,30 +35,24 @@ export default class ContactEditForm extends React.Component<Props> {
             return "";
         }
         const contactType = contactConfig.type;
-        if (contactType === ContactTypes.telegram) {
+        if (contactType.includes("telegram")) {
```

Comment:
Please, revert changes in `getPlaceholderForContactType`. With your remarks about `ContactTypes` uses I'll rewrite ToDo item about it. Give me some time